Rachis/Rachis/RaftEngineStatistics.cs

```diff
@@ -68,6 +73,16 @@ public void ReportCommitIndex(long index)
         }
     }
 
+    public class FollowerLastSentEntries
+    {
+        public FollowerLastSentEntries(ConcurrentDictionary<string, long> f2e)
+        {
+            FollowersToLastSent = f2e;
+        }
+        public readonly ConcurrentDictionary<string,long> FollowersToLastSent;
```

Comment:
it is meant to show what is now I know it may change and I don't care usually when I care about it the state is steady (usually stuck on a specific index)